domains/sajidhussain.json

```diff
@@ -0,0 +1,9 @@
+{
+    "owner": {
+        "username": "Sajid-tech",
+        "email": "s.khan9430319425@gmail.com"
+    },
+    "record": {
+        "URL": "sajidhussain.is-a.dev"
```

Comment:
Invalid URL record
- missing protocol
- you can't set the URl record as the domain itself

Reply:
I have updated my JSON file, review it!!